### DISCUSSION TITLE
SQL Functions Editor

### DISCUSSION THREAD

**Discussion by JrgMyr** [pawelsalawa/sqlitestudio#4205] [Q&A]
Where does the SQL Functions Editor in SQLiteStudio store its functions?

I would like to copy functions across different installations without retyping.

**Comment by pawelsalawa** [comment]
In the settings database. See https://github.com/pawelsalawa/sqlitestudio/wiki/Configuration_directory_location

**Reply by JrgMyr** [reply]
Oh yes, the config database. I went through it but have to say I did not find the function definitions, yet. Where should I look?

The config database would also be the natural place for an db-independant query store. See [issue #4065](https://github.com/pawelsalawa/sqlitestudio/issues/4065).

PS: Minor update for the wiki page. On my Windows system the path leading to the config file has one more folder than the [wiki page](https://github.com/pawelsalawa/sqlitestudio/wiki/Configuration_directory_location) mentions. There is an additional "/SalSoft" between %APPDATA% and "/SQLiteStudio/settings3".

**Reply by pawelsalawa** [reply]
Wiki page updated. Sorry it took so long. Thanks for spotting and reporting this!

**Reply by tuffnatty** [reply]
@pawelsalawa I think the wiki page should be updated, this time for the OS X part: on my system, the path is `/Users/USER_NAME/Library/Preferences/SalSoft/SQLiteStudio/settings3`, not what the wiki says.
While we're at it, i think there's an option to accept pull requests for the wiki?

**Reply by pawelsalawa** [reply]
Thanks @tuffnatty ! Updated.
